modules/articles/client/articles.client.module.js

```diff
@@ -1,4 +1,10 @@
-'use strict';
+(function (app) {
```

Comment:
I agree.. I'd prefer we don't use IIFE's like this. The way we're loading our client js files, is essentially the same behavior. This makes it harder to read, without a recognizable benefit.

I'm a huge fan of John Papa's angular style guide, but I don't think we have to take everything that he recommends.